backend/hostgw/hostgw.go

```diff
@@ -57,12 +57,13 @@ func New(sm subnet.Manager, network string) backend.Backend {
 	return b
 }
 
-func (rb *HostgwBackend) Init(extIface *net.Interface, extIP net.IP) (*backend.SubnetDef, error) {
+func (rb *HostgwBackend) Init(extIface *net.Interface, extIaddr net.IP, extEaddr net.IP) (*backend.SubnetDef, error) {
```

Comment:
Can you check if `extIaddr` and `extEaddr` differ and error out if so (saying it's not supported). I don't think this backend can work with NAT as it's all about L2.